internal/component/prometheus/remotewrite/types.go

```diff
@@ -298,6 +324,36 @@ func (a *AzureADConfig) Validate() error {
 		return fmt.Errorf("the provided Azure Managed Identity client_id provided is invalid")
 	}
 
+	// Validate OAuth if it is provided
```

Comment:
Is there a reason this is commented out?

Reply:
Was still rewriting them